config/webhook/manifests.yaml

```diff
@@ -83,6 +89,8 @@ webhooks:
         path: /validate-serving-kserve-io-v1alpha2-inferenceservice
     failurePolicy: Fail
     name: inferenceservice.kserve-webhook-server.validator
+    sideEffects: None
+    admissionReviewVersions: ["v1beta1"]
```

Comment:
here should be v1alpha2?

Reply:
`v1` version wouldn't allow `v1alpha2` for admissionReviewVersions.

Reply:
supported versions are `v1` and `v1beta1`.